<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDicomLocalStorageWidgetControls</class>
 <widget class="QWidget" name="QmitkDicomLocalStorageWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="localFilesGroupBox">
     <property name="title">
      <string>Local Dicom Storage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <property name="margin">
       <number>12</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="viewInternalDataButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>View</string>
           </property>
           <property name="icon">
            <iconset resource="../../../Plugins/org.mitk.gui.qt.dicom/resources/dicom.qrc">
             <normaloff>:/org.mitk.gui.qt.dicom/viewIcon.png</normaloff>:/org.mitk.gui.qt.dicom/viewIcon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../../../Plugins/org.mitk.gui.qt.datamanager/resources/datamanager.qrc">
             <normaloff>:/org.mitk.gui.qt.datamanager/Remove_48.png</normaloff>:/org.mitk.gui.qt.datamanager/Remove_48.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="QueryLayout" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ctkDICOMTableManager" name="ctkDicomBrowser" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMTableManager</class>
   <extends>QWidget</extends>
   <header>ctkDICOMTableManager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Plugins/org.mitk.gui.qt.datamanager/resources/datamanager.qrc"/>
  <include location="../../../Plugins/org.mitk.gui.qt.dicom/resources/dicom.qrc"/>
 </resources>
 <connections/>
</ui>
